framework/Models/ROM.py

```diff
@@ -348,6 +358,18 @@ def tryStrParse(s):
     self.mods = self.mods + list(set(utils.returnImportModuleString(inspect.getmodule(SupervisedLearning),True)) - set(self.mods))
     self.mods = self.mods + list(set(utils.returnImportModuleString(inspect.getmodule(LearningGate),True)) - set(self.mods))
 
+  def initialize(self,runInfo,inputs,initDict=None):
+    """
+      Method to initialize this class
+      @ In, runInfo, dict, it is the run info from the jobHandler
+      @ In, inputs, list, it is a list containing whatever is passed with an input role in the step
+      @ In, initDict, dict, optional, dictionary of all objects available in the step is using this model
+    """
+    # retrieve cross validation object
+    if self.cvInstance is not None:
+      self.cvInstance = self.retrieveObjectFromAssemblerDict('CV', self.cvInstance)
```

Comment:
? you overwrite the self.cvInstance?

Reply:
In _readMoreXml, self.cvInstance was first defined with cross validation post processor name. Here, we will use the name to retrieve the cross validation object.

Reply:
This makes sense. Note this may not always be ideal practice, since it's not repeatable; if you try to run "initialize" again with the already-initialized model, it will not work because cvInstance is no longer a string.

For now, this is okay.